### ISSUE TITLE
[Accessibility]: Tabbing through items in Revision Popover makes descriptions unreadable

### ISSUE THREAD

**Issue by plibither8** [sourcegraph/sourcegraph#37094]
### Audit type

Keyboard navigation

### Problem description

When opening the revision popover using the keyboard and trying to tab through the branches, the item descriptions to their right become unreadable (white-on-white), and only can be read when we hover over the item.

![Google Chrome 2022-06-13 at 13 18 26 000091](https://user-images.githubusercontent.com/31861755/173305444-ce0daabc-a7ae-4f41-95f7-f3829aa2085f.gif)

### Expected behavior

The link text color should not be white on focus, unless hovered over.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.
